pkg/apis/planetscale/v2/defaults.go

```diff
@@ -131,3 +120,45 @@ var defaultVitessImages = &VitessImages{
 
 	MysqldExporter: "prom/mysqld-exporter:v0.11.0",
 }
+
+var (
```

Comment:
I didn't know we assigned a priority class, do we evict non-Vitess related pods on scheduling failure?

Reply:
Yeah since the beginning we've been including a PriorityClass with the operator distro with a default priority value. Depending on the priority of other Pods in your system, this could cause Vitess Pods to preempt them. That'll still be the case by default, but at least with these new flags you can disable it.